lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -4028,15 +4052,20 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
         // because multiple accesses to the struct-held memory can require
         // different LSU parameters.
         addAnnotationDecorations(ResPtr, Decorations.MemoryAccessesVec);
-        if (allowDecorateWithBufferLocationINTEL(II))
+        if (allowDecorateWithBufferLocationOrLatencyControlINTEL(II)) {
           addAnnotationDecorations(ResPtr, Decorations.BufferLocationVec);
+          BM->addExtension(ExtensionID::SPV_INTEL_fpga_latency_control);
+          BM->addCapability(CapabilityFPGALatencyControlINTEL);
```

Comment:
No need to explicitly add extension and capability, you've done it implicitly by creating new map entries in  lib/SPIRV/libSPIRV/SPIRVDecorate.h and lib/SPIRV/libSPIRV/SPIRVEnum.h

Reply:
Infact, this caused test failures. I removed the statements. Thanks.